src/app/components/countries-list/countries-list.component.ts

```diff
@@ -279,7 +282,7 @@ export class CountriesListComponent implements OnInit {
     if (this.countryListForm.get('filterOptions').value) {
       this.commonSearchFilter();
     }
-    if ((value = '')) {
+    if ((value = '')) { // this will always be false, did you mean `if(value)`?
```

Comment:
whenever value is empty

Reply:
then it hsould be `value === ''` or `!value`

Reply:
ok